prow/oss/config.yaml

```diff
@@ -228,6 +228,16 @@ deck:
   hidden_repos:
   - looker
 
+slack_reporter_configs:
+  "*":
+    # Do not report any job types by default.
+    job_types_to_report:
+    # Do not report any job states by default.
+    job_states_to_report:
+    # By default reporting to the test channel, it is required so it has to be set here.
+    channel: channel-not-exist
```

Comment:
I think we've done something wrong if it is actually required to set all these fields to empty or fake values. This doesn't seem consistent with the way the rest of our config works. Can you explain why an empty config would be needed here?

Reply:
This was a lazy copy/paste from https://github.com/GoogleCloudPlatform/oss-test-infra/blob/f29b96f29b878d7192fa3d1e72125487619f9387/prow/knative/config.yaml#L48, I assume what Chi found there still applies.

wouldn't empty channel here result in slack report error in crier?